From cc057095ba4d4a9500163ec646a8bf134e9dd04a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 2 Dec 2022 22:49:41 +0100
Subject: [PATCH] place: Always center initial-setup/fedora-welcome

---
 src/core/place.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/core/place.c b/src/core/place.c
index 1075fe20d..35794ea6e 100644
--- a/src/core/place.c
+++ b/src/core/place.c
@@ -320,6 +320,10 @@ window_place_centered (MetaWindow *window)
 
   type = window->type;
 
+  if (g_str_equal (meta_window_get_wm_class (window), "org.gnome.InitalSetup") ||
+      g_str_equal (meta_window_get_wm_class (window), "fedora-welcome"))
+    return TRUE;
+
   return (type == META_WINDOW_DIALOG ||
     type == META_WINDOW_MODAL_DIALOG ||
     type == META_WINDOW_SPLASHSCREEN ||
-- 
2.38.1

