From b30a051cd190f06f0236339a5e24a391ce6f12ee Mon Sep 17 00:00:00 2001
From: GloriousEggroll <gloriouseggroll@gmail.com>
Date: Fri, 23 Jun 2023 10:26:17 -0600
Subject: [PATCH 2/3] 3085

---

diff --git a/src/backends/meta-backend.c b/src/backends/meta-backend.c
index e77b961..7e64c4f 100644
--- a/src/backends/meta-backend.c
+++ b/src/backends/meta-backend.c
@@ -787,6 +787,8 @@ meta_backend_constructed (GObject *object)
 
   g_assert (priv->context);
 
+  priv->settings = meta_settings_new (backend);
+
 #ifdef HAVE_LIBWACOM
   priv->wacom_db = libwacom_database_new ();
   if (!priv->wacom_db)
@@ -1181,7 +1183,6 @@ meta_backend_initable_init (GInitable     *initable,
   MetaBackend *backend = META_BACKEND (initable);
   MetaBackendPrivate *priv = meta_backend_get_instance_private (backend);
 
-  priv->settings = meta_settings_new (backend);
   priv->orientation_manager = g_object_new (META_TYPE_ORIENTATION_MANAGER, NULL);
 
   priv->monitor_manager = meta_backend_create_monitor_manager (backend, error);
-- 
2.41.0

